index.bs

```diff
@@ -1567,11 +1564,11 @@ dictionary MessageArgs {
 
 ### SIMID:Creative:requestChangeAdDuration ### {#simid-creative-requestChangeAdDuration}
 
-See <a href="#diagram-durationchange-known">Diagram 11. Known Ad Duration Change Sequence</a> and <a href="#diagram-durationchange-unknown">Diagram 12. Unknown Ad Duration Change Sequence</a>.
+See <a href="#diagram-durationchange-known">Diagram 10. Known Ad Duration Change Sequence</a> and <a href="#diagram-durationchange-unknown">Diagram 11. Unknown Ad Duration Change Sequence</a>.
```

Comment:
What do you think about not numbering the diagrams.  It seems like it is manual and error prone.  Perhaps the link and title are adquate. WDYT Andre?